src/ui/components/UserReview/styles.scss

```diff
@@ -16,15 +16,25 @@
   .TooltipMenu {
     width: fit-content;
   }
+
+  .Card-contents {
+    background: transparent;
+    padding: 0;
+  }
+
+  .Card.ShowMoreCard.UserReview-body.UserReview-emptyBody {
```

Comment:
This seems like a lot of selectors. Do we really need all of them to get the appropriate level of specificity?

Reply:
No, i reduced it to 2 selectors